src/styles/partials/desktop/_recapitulatif.scss

```diff
@@ -0,0 +1,42 @@
+.recapitulatif-form {
+  .chapter-block:not(:last-child) {
+    margin-bottom: 1.25rem;
```

Comment:
there is a mix between rem and px
what's the standard ?

Reply:
En effet il est pertinent de conserver les mêmes règles partout.
Je vais changer ça en px.
Je n'ai pas de standard à ma connaissance :/